recipe-client-addon/lib/EventEmitter.jsm

```diff
@@ -20,15 +19,13 @@ this.EventEmitter = function(driver) {
       Promise.resolve()
         .then(() => {
           if (!(eventName in listeners)) {
-            driver.log(`EventEmitter: Event fired with no listeners: ${eventName}`);
+            log.info(`EventEmitter: Event fired with no listeners: ${eventName}`);
             return;
           }
-          // freeze event to prevent handlers from modifying it
-          const frozenEvent = Object.freeze(event);
```

Comment:
> But that one won't have the handlers of the one inside the sandbox, right?

It will, because the handlers aren't a property of the emitter, but instead are in the scope of the constructor function, and the docs on [`exportFunction`](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Language_Bindings/Components.utils.exportFunction) say that the cloned function has access to its surrounding closure.